pyccl/tests/test_pt_power.py

```diff
@@ -233,3 +234,26 @@ def test_translate_IA_norm_raises():
     with pytest.raises(ValueError):
         c_1, c_d, c_2 = ccl.nl_pt.translate_IA_norm(COSMO, ZZ2, a1=a_1_v,
                                                     Om_m2_for_c2=False)
+                                                    
+def test_return_ptc():
```

Comment:
If this passes and flake8 doesn't complain, we can still merge this even if travis complains